docs/faq.rst

```diff
@@ -149,9 +147,9 @@ flag. For more information, see :ref:`common-configs`.
 How do I use Avro data and integrate with Confluent Schema Registry?
 ====================================================================
 
-Configure the ``ksql.schema.registry.url`` to point to Schema Registry (see :ref:`common-configs`).
+Configure the ``ksql.schema.registry.url`` to point to Schema Registry (see :ref:`install_ksql-avro-schema`).
```

Comment:
Nit: Configure the ``ksql.schema.registry.url`` *in the KSQL server configuration* to point to...